### ISSUE TITLE
Contrast between copy icons and background is not high enough

### ISSUE THREAD

**Issue by szakarias** [dart-lang/pub-dev#7427]
We could consider making them black.

**Comment by sigurdm** [comment]
![image](https://github.com/dart-lang/pub-dev/assets/8613953/3bbe6526-608f-43ae-94c5-87c606a226ca)
